app/react/utils/specs/api.spec.js

```diff
@@ -150,6 +172,21 @@ describe('api', () => {
       });
     });
 
+    describe('when request return a server error of too large request', () => {
+      it('should notify the error with this message', async () => {
+        const requestParams = new RequestParams({ key: 'test' }, { header: 'value' });
+        try {
+          await api.post('validation_error', requestParams);
+          fail('should throw error');
+        } catch (e) {
+          testNotificationDisplayed(
+            "validation failed:  .metadata['prop1'] should be string, .metadata['prop2'] is too longer,",
```

Comment:
Is there no way to differentiate that it, indeed, was a string, but it is too large?  This may mislead into interpreting it was not a string.